src/schema/v2/artwork/artworkConsignmentSubmissionType.ts

```diff
@@ -86,28 +86,84 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
         description: "Submission state label visible to the user.",
         resolve: ({ state }) => {
           switch (state.toLowerCase()) {
+            case "draft":
+            case "hold":
+            case "closed":
+              return null
             case "approved":
+              return "Approved"
             case "rejected":
-            case "closed":
-            case "published":
-              return "Evaluation Complete"
+              return "Submission Unsuccessful"
             default:
               return "In Progress"
           }
         },
       },
+      stateLabelColor: {
+        type: GraphQLString,
+        resolve: ({ state }) => {
+          switch (state.toLowerCase()) {
+            case "rejected":
+              return "black60"
+            default:
+              return "black100"
+          }
+        },
+      },
```

Comment:
What is the `stateLabelColor` for? Is it the button? Maybe for the button we could define the color like this:

```suggestion
      buttonVariant: {
        type: GraphQLString,
        resolve: ({ state }) => {
          switch (state.toLowerCase()) {
            case "approved":
            case "draft":
              return "primaryBlack"
            default:
              return "secondaryBlack"
          }
        },
      },
```

Reply:
It's the color for `stateLabel`, it's black100 for all statuses and only for Submission Unsuccessful (rejected state) it's black60
Not sure if it's an overkill or not